Source/WebCore/platform/graphics/cg/ImageDecoderCG.cpp

```diff
@@ -660,6 +659,7 @@ void ImageDecoderCG::setData(const FragmentedSharedBuffer& data, bool allDataRec
     CGImageSourceUpdateData(m_nativeDecoder.get(), contiguousData->createCFData().get(), allDataReceived);
     
     m_uti = decodeUTI(contiguousData.get());
+    m_decodeMaskImage = m_uti == "public.xbitmap-image"_s;
```

Comment:
`m_isXBitmapImage` might be a clearer name.

Reply:
Sure.